src/citrine/resources/data_concepts.py

```diff
@@ -79,56 +85,68 @@ def build(cls, data: dict, session: Session = None):
             An object corresponding to a data concepts resource.
 
         """
+        data_copy = deepcopy(data)
+        # Extract the values that are only for the client.
+        # They will be attached to the deserialized object later.
+        client_only_dict = dict()
+        for client_key in cls._client_keys:
+            val = DataConcepts._pop_field(data_copy, client_key)
```

Comment:
Why did you have to define `_pop_field`? The type hint on this method suggests (and I would expect) that `data` is a `dict`.

Reply:
Also is there a reason you're using `DataConcepts` instead of `cls` in this method? AFAIK the difference between the two is that `DataConcepts.foo` ignores any overrides to `foo` in a child class.

Reply:
In the recursive dance that occurs when building an object with soft links, `build` can get called with `data` as an object. Hence the need for `_pop_field`

Reply:
I used `DataConcepts` because I intended `_pop_field` to be a static method that doesn't get overridden, so figured `DataConcepts` and `cls` both work equally well, but `DataConcepts` tells the reader where to find the method.